common/scylla-image-post-start.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Scylla cloud image post start service
+After=scylla-server.service
+ConditionPathExists=!/etc/scylla/machine_image_post_start
+
+[Service]
+Type=oneshot
+# The ExecStartPre makes sure the scylla_post_start.py script 
+# will run only after scylla-server is up and running
+ExecStartPre=/bin/sleep 30
+ExecStart=/opt/scylladb/scylla-machine-image/scylla_post_start.py
```

Comment:
Do we required root privilege for post_start script?
If not, it's better to use `User=scylla` to run the script as a non-root user.
It feels like safer since we run shellscript received from external place.